scripts/bootstrap.sh.in

```diff
@@ -93,7 +93,7 @@ configure_yum_local_repository() {
 
     gpg_keys=$(
         find "$repo_path" -maxdepth 1 -name "RPM-GPG-KEY-*" \
-	    -printf "file://%p "
+        -printf "file://%p "
```

Comment:
The `-printf` is an argument to the `find`, so IMHO it makes sense to be more indented.

Reply:
Yes indeed, my bad (replaced a tab with spaces)